packages/tools/src/utilities/math/basic/BasicStatsCalculator.ts

```diff
@@ -14,6 +14,8 @@ export default class BasicStatsCalculator extends Calculator {
   // Values for Welford's algorithm
   private static runMean = [0];
   private static m2 = [0];
+  private static squaredDiffSum = [0];
```

Comment:
@wayfarer3130 can you double check this file only? i think the rest is good to go